packages/material-ui-icons/package.json

```diff
@@ -1,9 +1,9 @@
 {
   "name": "material-ui-icons",
-  "private": true,
+  "private": false,
   "author": "Material-UI Team",
   "version": "1.0.0-beta.17",
-  "description": "Material Design Svg Icons converted to Material-UI React components.",
+  "description": "Material-UI Design Svg Icons converted to Material-UI React components.",
```

Comment:
That was correct - They're Material Design's icons (could perhaps be shortened to just 'Material Icons' though)

Reply:
Ok, I will revert